mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -108,7 +113,7 @@ def __repr__(self) -> str:
         )
         return f"{self.name}({opt_targets},config={self._config})"
 
-    def __enter__(self) -> 'Optimizer':
+    def __enter__(self) -> "Optimizer":
```

Comment:
There's a whole bunch of style-only changes in here that makes reviewing a little difficult.

Can you please revert those, or separate them out to a different PR?